src/atmos_param/diffusivity/diffusivity.F90

```diff
@@ -496,6 +498,18 @@ subroutine diffusivity_pbl(t, u, v, z_half, h, u_star, b_star, &
     k_m(:,:,k) = k_m_ref*factor
     k_t(:,:,k) = k_t_ref*factor
   end where
+
+
+  if(use_pog_bug_fix) then
```

Comment:
I have added the namelist option to turn the fix on and off, so that old-behaviour can be retained if desired.